example/example.ts

```diff
@@ -1,4 +1,8 @@
+/* this file doesn't work, but here we are */
```

Comment:
Remove this comment as this import is not expected to work

Reply:
I fully expected it to work